app/src/components/Button.tsx

```diff
@@ -8,25 +8,34 @@ type Props = {
   onClick?: () => void
 }
 
-const Button = (props: Props): ReactElement => {
-  const { disabled, labelKey, style, onClick } = props
+// Next.js requires forwarding refs if this functional component might
+// be a child component of next/link. For more info, see
+// https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-functional-component
+const Button = React.forwardRef(
+  (props: Props, ref: React.LegacyRef<HTMLButtonElement>): ReactElement => {
+    const { disabled, labelKey, style, onClick } = props
```

Comment:
@karinamzalez This change fixes all of the warnings we have been seeing about `forwardRef(LinkComponent)`